README.rst

```diff
@@ -76,18 +76,31 @@ than one CPU core, refer to our `parallel backend guide`_.
 
 **Note for Windows users**
 
-The pip installer for Neuron does not yet work for Windows. In this case, it is better to
-install ``hnn_core`` without the dependencies::
+The pip installer for Neuron does not yet work for Windows. In this case we recommend
+installing `WSL <https://docs.microsoft.com/en-us/windows/wsl/install>`_ on your
```

Comment:
It seems that it's not obvious Anaconda needs to be installed also on WSL. Clarifying may help!